config/lib/express.js

```diff
@@ -40,8 +40,9 @@ module.exports.initLocalVariables = function (app) {
 
   // Passing the request url to environment locals
   app.use(function (req, res, next) {
-    res.locals.host = req.protocol + '://' + req.hostname;
-    res.locals.url = req.protocol + '://' + req.headers.host + req.originalUrl;
+    var domain = config.app.domain || req.headers.hostname;
```

Comment:
Shouldn't we use `req.hostname`, so we don't have to worry about the port? We can get the port from the config. Unless there's something that I'm unaware of, this would fix the issue you encountered with *livereload*